llbuild.podspec

```diff
@@ -0,0 +1,88 @@
+Pod::Spec.new do |s|
+
+  s.name         = "llbuild"
+  s.version      = ENV['LLBUILD_PODSPEC_VERSION'] || "9999.0.0"
+  s.summary      = "A low-level build system."
+
+  s.description  = <<-DESC.strip_heredoc
+                    **llbuild** is a set of libraries for building build systems. Unlike most build
+                    system projects which focus on the syntax for describing the build, llbuild is
+                    designed around a reusable, flexible, and scalable general purpose *build
+                    engine* capable of solving many "build system"-like problems. The project also
+                    includes additional libraries on top of that engine which provide support for
+                    constructing *bespoke* build systems (like `swift build`) or for building from
+                    Ninja manifests.
+                   DESC
+
+  s.homepage     = "https://github.com/apple/swift-llbuild"
+  s.license      = { type: 'Apache 2.0', file: "LICENSE.txt" }
+
+  s.documentation_url  = "https://llbuild.readthedocs.io/"
+  s.author             = "Apple"
+
+  s.ios.deployment_target     = "10.0"
```

Comment:
Please make sure the podspec and SPM deployment targets are the same

Reply:
After removing the code that ITMS was complaining about, minimum version can be 9.